internal/controller/core/suite_test.go

```diff
@@ -60,6 +61,9 @@ var _ = BeforeSuite(func() {
 	err = v1alpha1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+	err = nxosv1alpha1.AddToScheme(scheme.Scheme)
```

Comment:
We should not leak this provider api group into the "core". The core should be standalone, with no dependency on a provider-specific api group.